VALUES.md

```diff
@@ -0,0 +1,30 @@
+# Ayo.js Community Values
+
+We don't care how you pronounce it: be it "ey-yo", "I-oh", or "Awooooo" -- the Ayo.js project strives to create an environment where you can feel at home and focus on your contributions to this nice shared space that we all benefit from.
```

Comment:
done and done

(thanks @hypesystem for coming with actual suggestions for rewording!)